kable-api-v1.yaml

```diff
@@ -1415,6 +1415,307 @@ paths:
           description: "invalid input, object invalid"
         "401":
           description: "unauthorized"
+  /bundles:
+    get:
+      tags:
+        - bundles
+      summary: get all bundles
+      operationId: getAllBundles
+      description: |
+        Retrieve all bundles.
+      parameters:
+        - in: header
+          name: Kable-Client-Id
+          description: Your client ID, found in the dashboard of your Kable account.
+          schema:
+            type: string
+          example: kci_3c90e9ac92c64f31ae8ed84d21e18740
+          required: true
+        - in: header
+          name: Kable-Client-Secret
+          description: Your `LIVE` or `TEST` secret key. Customers exist across all environments, so it does not matter which environment you use to fetch customers.
```

Comment:
`description: Your `LIVE` or `TEST` secret key.`

(You can drop the bit about customers)

Reply:
This comment applies throughout this PR -- same edit to each method